src/aleph/web/controllers/posts.py

```diff
@@ -85,10 +85,7 @@ async def view_posts_list(request):
     context = {"posts": posts}
 
     if pagination_per_page is not None:
-        total_msgs = await Message.collection.count_documents(
-            filter=find_filters,
-            projection={"_id": 0},
```

Comment:
Is the `_id` field still removed from the API ? If so, where is it filtered out ?

Reply:
In the aggregate query directly.